src/components/atoms/OcProgress/OcProgress.vue

```diff
@@ -30,7 +34,11 @@ export default {
      */
     max: {
       type: Number,
-      required: true,
+      required: false,
+      default: undefined,
+      validator: value => {
+        return value >= 0
```

Comment:
Doesn't behave correctly:
- `undefined` is valid
- 0 is invalid

Reply:
Huh? `undefined > 0` evaluates to `false` for me

Reply:
> maybe it doesn't trigger the validator at all since it's the default. that might be...

Yes that's what I meant :D The check itself would return `false` of course